cms/envs/devstack.py

```diff
@@ -39,7 +39,7 @@
 
 ################################# LMS INTEGRATION #############################
 
-LMS_BASE = "localhost:8000"
+# LMS_BASE = "localhost:8000"  # Edraak: Commented out because we use domain names locally.
```

Comment:
And what the base should look like? Isn't it better to get it from the envs file rather than commenting it out?